portal-web/docroot/html/taglib/ui/page_iterator/start.jsp

```diff
@@ -413,34 +445,38 @@ NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
 						</c:if>
 
 						<li class="page-item">
-							<a class="page-link" href="<%= _getHREF(formName, namespace + curParam, pages, jsCall, url, urlAnchor) %>" onclick="<%= forcePost ? _getOnClick(namespace, curParam, pages) : "" %>"><span class="sr-only"><liferay-ui:message key="page" /><%= StringPool.NBSP %></span><%= pages %></a>
+							<liferay-ui:csp>
+								<a class="page-link" href="<%= _getHREF(formName, namespace + curParam, pages, jsCall, url, urlAnchor) %>" onclick="<%= forcePost ? _getOnClick(namespace, curParam, pages) : "" %>"><span class="sr-only"><liferay-ui:message key="page" /><%= StringPool.NBSP %></span><%= pages %></a>
+							</liferay-ui:csp>
 						</li>
 					</c:otherwise>
 				</c:choose>
 
 				<li class="page-item <%= (cur < pages) ? StringPool.BLANK : "disabled" %>">
-					<c:choose>
-						<c:when test="<%= cur < pages %>">
-							<a class="lfr-portal-tooltip page-link" href="<%= _getHREF(formName, namespace + curParam, cur + 1, jsCall, url, urlAnchor) %>" onclick="<%= forcePost ? _getOnClick(namespace, curParam, cur + 1) : "" %>" title="<%= LanguageUtil.get(request, "next-page") %>">
-						</c:when>
-						<c:otherwise>
-							<div class="page-link">
-						</c:otherwise>
-					</c:choose>
-
-						<liferay-ui:icon
-							icon='<%= PortalUtil.isRightToLeft(request) ? "angle-left" : "angle-right" %>'
-							markupView="lexicon"
-						/>
-
-					<c:choose>
-						<c:when test="<%= cur < pages %>">
-							</a>
-						</c:when>
-						<c:otherwise>
-							</div>
-						</c:otherwise>
-					</c:choose>
+					<liferay-ui:csp>
+						<c:choose>
+							<c:when test="<%= cur < pages %>">
+								<a class="lfr-portal-tooltip page-link" href="<%= _getHREF(formName, namespace + curParam, cur + 1, jsCall, url, urlAnchor) %>" onclick="<%= forcePost ? _getOnClick(namespace, curParam, cur + 1) : "" %>" title="<%= LanguageUtil.get(request, "next-page") %>">
```

Comment:
![image](https://github.com/user-attachments/assets/c31181f5-105a-4ad9-8fc8-1e3e1f3dce92)

Reply:
For the record: it appears that this started to work due to some cache being refreshed after the proposed change more than because of the change itself. So this PR is safe.